run-e2e-tests.js

```diff
@@ -0,0 +1,120 @@
+const http = require('http');
+const fs = require('mz/fs');
+const express = require('express');
+const seleniumAssistant = require('selenium-assistant');
+const Mocha = require('mocha');
+
+function browserFilter(browser) {
+  return browser.getReleaseName() === 'stable'
+    && ['chrome'].includes(browser.getId());
+}
+
+function startServer() {
+  return new Promise(resolve => {
+    const app = express();
+    app.use(express.static('docs'));
+    const server = http.createServer(app).listen();
+    server.on('listening', _ => resolve(server));
+  });
+}
+
+async function buildMocha() {
+  const mocha = new Mocha();
+  const elements = await fs.readdir('elements');
+  const filteredElements = await Promise.all(
+    elements.map(async element => {
+      const files = await fs.readdir(`elements/${element}/`);
+      if (files.some(f => /\.e2etest\.js/.test(f)))
+        return element;
+      else
+        return '';
+    })
+  );
+  filteredElements
+    .filter(name => !!name)
+    .forEach(name =>
+      mocha.addFile(`./elements/${name}/${name}.e2etest`));
+  mocha.suite.timeout(60000);
+  return new Promise(resolve => mocha.loadFiles(resolve))
+    .then(_ => mocha);
+}
+
+function unsandboxChrome(browser) {
+  const isChrome = browser.getReleaseName() === 'stable'
+    && ['chrome'].includes(browser.getId());
+  if (!isChrome) return browser;
+  browser
+    .getSeleniumOptions()
+    .addArguments('--no-sandbox');
```

Comment:
What's the need for unsandboxing chrome? Is it to run ajax against the file system?